webclient/components/ptinfo-single/1time-Mrow-mField/diagnosis/db/client-side/structure/master-table-of-diagnosis.js

```diff
@@ -10,6 +10,8 @@ const intUniqueId = () => ++count
 export default class clientSideTblMasterDiagnosis extends clientSideTableManage {
   static entity = 'tableMasterOfDiagnosis'
 
+  static apiUrl = 'http://localhost:8000/public/api/diagnosis/v20'
```

Comment:
Why you store apiUrl in 2 places. This should be only in patient-table-of-diagnosis.js

Reply:
Done